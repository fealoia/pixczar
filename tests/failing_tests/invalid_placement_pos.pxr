Void main() {
	Pix tri = new Pix();
  	tri.makeTriangle(200,[200,200,200]);
  	Int x = 3;
	Placement placed1 = new Placement(tri,200,300);
	Placement placed2 = new Placement(x,300,300);
}
