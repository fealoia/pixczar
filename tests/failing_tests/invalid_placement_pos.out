Fatal error: exception Failure("illegal argument found Int expected Pix in x")
